packages/backend/src/apps/custom-api/actions/http-request/index.ts

```diff
@@ -97,15 +100,28 @@ const action: IRawAction = {
       if (err.message === RECURSIVE_WEBHOOK_ERROR_NAME) {
         throw new StepError(
           RECURSIVE_WEBHOOK_ERROR_NAME,
-          RECURSIVE_WEBHOOK_ERROR_SOLUTION,
+          'Ensure that you are not redirecting back to a plumber URL.',
+          $.step.position,
+          $.app.name,
+        )
+      }
+
+      if (err.message === DISALLOWED_IP_RESOLVED_ERROR) {
```

Comment:
suggestion for future: comparing error messages is quite brittle, it could be better to use a new error class that inherits from StepError, and perform the comparison via _instanceof_ instead

Reply:
cant think of any other way to do it right now because all errors thrown in interceptors get wrapped in the generic HttpError.